src/main/java/com/google/alpollo/helpers/AnalysisHelper.java

```diff
@@ -128,4 +127,27 @@ public static String getInterpretation(double score, double magnitude) {
 
     return MIXED;
   }
+
+  public static List<SongEntity> getFilteredTop10(List<SongEntity> list) {
```

Comment:
nit: maybe change to `getFilteredTopSongs`, in case `MAX_ENTITIES` values change in the future.